ui/app/scripts/stream/services/metamodel.js

```diff
@@ -176,7 +176,7 @@ define(function (require) {
                                             properties[p.id] = p;
                                         }
                                     }
-                                    if (p.sourceType === 'org.springframework.cloud.stream.app.transform.ProgrammableRxJavaProcessorProperties') {
+                                    if (p.sourceType === 'org.springframework.cloud.stream.app.transform.ProgrammableReactiveProcessorProperties') {
```

Comment:
We looked at this repo https://github.com/markfisher/programmable-reactive-processor As far as I understand this is going to be in the app-starters soon. This is just a preparation commit, thus RxJava processor can be tested with Flo right away